src/debug/HyprCtl.cpp

```diff
@@ -1261,6 +1261,11 @@ std::string dispatchSeterror(eHyprCtlOutputFormat format, std::string request) {
     return "ok";
 }
 
+std::string dispatchSetProp(eHyprCtlOutputFormat format, std::string request) {
+    auto result = g_pKeybindManager->m_mDispatchers["setprop"](request.substr(request.find_first_of(' ') + 1, -1));
+    return "DEPRECATED: use hyprctl dispatch setprop instead" + (result.success ? "" : "\n" + result.error);
```

Comment:
worth mentioning it's a dispatcher if someone is running `,exec,hyprctl setprop...`

"use the setprop dispatcher or hyprctl dispatch setprop instead"

Reply:
sure, but if they're running that they won't see the output anyways

Reply:
right, disregard then